src/libmongoc/src/mongoc/mongoc-uri.c

```diff
@@ -2782,6 +2784,7 @@ mongoc_uri_set_option_as_int64 (mongoc_uri_t *uri,
 
    r = _mongoc_uri_set_option_as_int64_with_error (uri, option, value, &error);
    if (!r) {
+      printf ("got a warning for int64\n");
```

Comment:
Did you mean to leave this print statement here?

Reply:
Nope, removed